docs/changelog/89292.yaml

```
@@ -0,0 +1,5 @@
+pr: 89292
+summary: move log-related logic into log block in IndexLifecycleRunner
+area: ILM
```

This needs to conform to a specific enum to pass CI, so can you change this to:

```suggestion
area: ILM+SLM
```